packages/api-nest/src/library/library.resolver.ts

```
@@ -145,6 +220,28 @@ export class LibraryResolver {
+    description: 'Update library item metadata (title, author, description)',
+  })
+  @UseGuards(JwtAuthGuard)
+  async updateLibraryItem(
+    @CurrentUser() user: User,
+    @Args('id', { type: () => String, description: 'Library item ID' })
+    id: string,
+    @Args('input', {
+      type: () => UpdateLibraryItemInput,
+      description: 'Updated library item metadata',
+    })
+    input: UpdateLibraryItemInput,
```

ESLint

[error] Delete `,`

(prettier/prettier)